.rustfmt.toml

```diff
@@ -0,0 +1,6 @@
+unstable_features = true
+indent_style = "Block"
```

Comment:
This is the default, right? So we can avoid setting it here?

Reply:
That's correct. The config file is now empty, so I'm removing it.